de/privacy/current.md

```diff
@@ -245,19 +245,29 @@ Diese Webseite nutzt die Google Cloud Plattform, ein Cloud Computing Service der
-Wenn Du dich dazu entscheidest, einen Spendenbeitrag über unser Spendenformular zu leisten, werden die durch das Spendenformular erfasste Daten auf Highrise gespeichert. Highrise ist ein Customer-Relationship-Management Dienst welcher in den USA ansässig ist. Die gespeicherten Daten umfassen:
+Wenn Du mit uns Kontakt aufnimmst, oder dich dazu entscheidest, einen Spendenbeitrag über unser Spendenformular zu leisten, werden die durch das Spendenformular erfasste Daten auf Hubspot gespeichert. Hubspot ist ein Customer-Relationship-Management Dienst welcher in den USA ansässig ist. Die gespeicherten Daten umfassen:
 
 - Deine Kontaktdaten,
 - die Höhe und den Rhythmus Deiner Spende
 - seit wann Du spendest.
 
-Highrise nutzt diese Daten nicht und gibt sie grundsätzlich auch nicht weiter. Highrise gibt gespeicherte Daten nur weiter, (i) mit Einwilligung seitens Serlo.org, (ii) um gesetzlichen Verpflichtungen nachzukommen oder (iii) im Falle eines Verkaufs des Unternehmens, einer Fusion mit einem anderen Unternehmen oder ähnlichen geschäftlichen Interaktionen.
+Hubspot nutzt diese Daten nicht und gibt sie grundsätzlich auch nicht weiter. Hubspot gibt gespeicherte Daten nur weiter, (i) mit Einwilligung seitens Serlo.org, (ii) um gesetzlichen Verpflichtungen nachzukommen oder (iii) im Falle eines Verkaufs des Unternehmens, einer Fusion mit einem anderen Unternehmen oder ähnlichen geschäftlichen Interaktionen.
 
-Highrise ist unter dem EU-US Datenschutzabkommen zertifiziert und damit verpflichtet, den EU-Datenschutzvorgaben nachzukommen. Des weiteren hat Serlo.org mit Highrise ein Data-Processing Agreement abgeschlossen, wodurch sich Highrise verpflichtet, die Daten unserer Nutzer zu schützen und sie nicht weiterzugeben. Die Verarbeitung erfolgt auf Grundlage des Art. 6 Abs. 1 lit. a DSGVO (Einwilligung) und Art. 6 Abs. 1 lit f (Berechtigtes Interesse). Du kannst die erteilte Einwilligung zur Speicherung der Daten jederzeit widerrufen. Die Rechtmäßigkeit der bereits erfolgten Datenverarbeitungsvorgänge bleibt vom Widerruf unberührt.
+Die Verarbeitung erfolgt auf Grundlage des Art. 6 Abs. 1 lit. a DSGVO (Einwilligung) und Art. 6 Abs. 1 lit f (Berechtigtes Interesse). Du kannst die erteilte Einwilligung zur Speicherung der Daten jederzeit widerrufen. Die Rechtmäßigkeit der bereits erfolgten Datenverarbeitungsvorgänge bleibt vom Widerruf unberührt.
 
-Die Datenschutzerklärung von Highrise findest Du [hier](https://highrisehq.com/privacy/).
+Die Datenschutzerklärung von Hubspot findest Du [hier](https://legal.hubspot.com/de/privacy-policy).
+
+#### Planetscale
+
+Diese Webseite nutzt den Dienst [Planetscale](https://planetscale.com/). Der Anbieter ist PlanetScale Inc. 535 Mission St 14th floor, San Francisco, CA 94105, United States. Planetscale ist eine moderne Datenbank auf der wir einige Daten speichern um unsere Webseite bereitzustellen. Einige deiner Daten können bei deinem Besuch unserer Webseite an Planetscale übermittelt und dort gespeichert werden.
```

Comment:
> Einige deiner Daten können bei deinem Besuch unserer Webseite an Planetscale übermittelt und dort gespeichert werden

It would be good to specify what data are stored, otherwise it is very vague and could make users worried.
As far as I know no personal information is stored.

Besides that, we could start justifying why we use this service, saying something like 'Some analytics info are too complex to be handled with Simple Analytics, that is why we use Planetscale...'